go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/grafana/terraform-provider-grafana
 
 go 1.18
 
+replace github.com/grafana/grafana-api-golang-client => github.com/grafana/grafana-api-golang-client v0.14.1-0.20221205183704-2e3d18a0df20
```

Comment:
nit: should probably just bump the version once the next API client is released?

Reply:
Yup, this was just a temporary replace to test out the client